lib/template.html

```diff
@@ -51,7 +51,11 @@
 
 {% if site.twitter %}
   {% if seo_tag.image %}
-    <meta name="twitter:card" content="summary_large_image" />
+      {% if site.twitter.smallSummary %}
```

Comment:
I like that idea!
Is there a way to do a double default.
E.g.
Page.twitter.card
Then
Site.twitter.card
Then
Summary large image